paginate.md

```diff
@@ -0,0 +1,167 @@
+# Adding Pagination
+
+Using the pagination library, [will_paginate](https://github.com/mislav/will_paginate), we will add pagination to our `books` `index` view so we don't create an infinite scrolling scenario.
+
```

Comment:
Might be worth noting that this is only one pagination option. Trestle already shipped with `kaminari`. And I've been using `pagy` lately.